aries_vcx/src/common/credentials/encoding.rs

```diff
@@ -299,35 +365,37 @@ pub mod unit_tests {
             }
         });
 
-        static TEST_CREDENTIAL_DATA: &str = r#"{"address2":["101 Wilson Lane"],
+        let attr_data = CredentialAttribute::new(
```

Comment:
This doesn't seem to make quite sense, `CredentialAttribute` as I understand currently refers to 1 attribute (with raw and encoded value), but here you are passing entire json of attributes and their respective raw values.

Reply:
You're right, and my apologies on this one. I mixed things here. I will get that fixed.